app/models/setting/ansible.rb

```diff
@@ -90,6 +90,16 @@ def load_defaults
               false,
               format(N_('%{cfgmgmt} out of sync disabled'),
                      :cfgmgmt => 'Ansible')
+            ),
+            set(
+              'ansible_implementation',
+              N_('Foreman will run Ansible playbooks using this implementation'),
```

Comment:
Would be good to mention possible values for this setting in description

Reply:
Or is the `:collection` supposed to be shown somewhere? I don't see it on my devel setup